src/PostgrestClient.ts

```diff
@@ -2,12 +2,20 @@ import PostgrestQueryBuilder from './PostgrestQueryBuilder'
 import PostgrestFilterBuilder from './PostgrestFilterBuilder'
 import PostgrestBuilder from './PostgrestBuilder'
 import { DEFAULT_HEADERS } from './constants'
-import { Fetch } from './types'
+import { Fetch, GenericSchema } from './types'
 
-export default class PostgrestClient {
+export default class PostgrestClient<
+  Database = any,
```

Comment:
Guessing we'll have to live with these two `any` types because `unknown` would limit the operations we can perform on `Database` ?

Reply:
yeah, it's an escape hatch to ease the transition to the new typings